Home_10/Task_10_1/src/Util.java

```diff
@@ -0,0 +1,152 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Util { //тут творится чертовщина - я чуть голову не сломал, пока осознал, что наворотил в прошлый раз в этой задаче
+    public static final Scanner scanner = new Scanner(System.in);
+    public static final BufferedReader bfr = new BufferedReader(new InputStreamReader(System.in));
+
+    public static void print(final String print) { //просто метод красоты, чтобы меньше писать в мейне
+        System.out.println(print);
+    }
+
+    public static int askForInteger() {
+        System.out.println("Enter count of your students (use only digits 0-9)");
+        while (true) { //запускаю цикл - пока пользователь не введет корректные данные, программа его не отпустит
+            if (!scanner.hasNextInt()) { //проверяю корректность данных - если введена НЕ цифра, вывожу сообщение об ошибке и цикл с вводом запускается снова
+                System.out.println("Use only numbers 0-9.");//вывод сообщения с ошибкой
+                scanner.next(); //пренебрегаю некорректным символом и перехожу к проверке следующего ввода
+            } else { //если пользователь все заполнил правильно, возвращаю int-значение
+                return scanner.nextInt();  //возврат корректного целочисленного значения для сравнения в main
+            }
+        }
+    }
+
+    public static ArrayList<ArrayList<String>> fillAttendSheet(final int count) { // метод заполнения таблицы
+        int localCount = count + 1; //мы получаем число студентов
+        ArrayList<ArrayList<String>> arrayLists = new ArrayList<>();
```

Comment:
Здесь и далее лучше оперировать абстракциями(интерфейсами), а не реализациями
```suggestion
        List<List<String>> arrayLists = new ArrayList<>();
```

Reply:
Да, так все советуют. Не знаю почему я делаю конкретные реализации. Возможно, какая-то вредная привычка. Буду стараться отучить себя. Спасибо, что написали об этом!